client/src/components/RecipeCard.js

```diff
@@ -167,4 +175,32 @@ const Card = styled.div`
     width: 80%;
     z-index: -1;
   }
+
+  .font-upload:hover {
+    color: red;
+  }
+
+  .font-upload {
+  }
+`;
+
+const Button = styled.button`
+  .btn {
+    background-color: DodgerBlue;
+    border: none;
+    color: white;
+    padding: 12px 16px;
+    font-size: 16px;
+    cursor: pointer;
+  }
+
+  background-color: #bebebe;
+  border: 2px solid #fff;
+  border-radius: 20px;
+  color: #922c88;
+  font-weight: bold;
+  padding: 1.2rem;
+  opacity: 0.7;
+  box-shadow: 0 0 5px rgba(75, 75, 75, 0.07);
+  text-transform: uppercase;
```

Comment:
Why do you use the additonal class `.btn` here inside your styled component? 
This is really confusing. Consider to either have two different styled button components or use props to dynamically change the designs that should defer.